BlockSettleUILib/UiUtils.cpp

```diff
@@ -612,14 +612,21 @@ QString UiUtils::NormalizeString(const QString& value)
 
 QValidator::State UiUtils::ValidateDoubleString(QString &input, int &pos, const int decimals)
 {
-   if (input.isEmpty()) {
-      return QValidator::Acceptable;
-   }
-
    static const QChar defaultDecimalsSeparatorChar = QLatin1Char('.');
 
    QString tempCopy = UiUtils::NormalizeString(input);
 
+   QStringList list = tempCopy.split(defaultDecimalsSeparatorChar);
+   if (list.size() == 2 && list[1].length() > decimals && pos == input.length()) {
+      list[1].resize(decimals);
+      input = list[0] + QLatin1Char('.') + list[1];
+      tempCopy = input;
+   }
+
+   if (input.isEmpty()) {
```

Comment:
Not sure why this moved here. There is no point to split or do any validations if input is empty

Reply:
Sure, fixed.